test-clients/src/main/java/com/hedera/services/bdd/suites/file/DiverseStateCreation.java

```diff
@@ -0,0 +1,199 @@
+import static com.hedera.services.bdd.spec.transactions.TxnVerbs.contractCall;
+import static com.hedera.services.bdd.spec.transactions.TxnVerbs.contractCreate;
+import static com.hedera.services.bdd.spec.transactions.TxnVerbs.fileCreate;
+import static com.hedera.services.bdd.spec.transactions.TxnVerbs.fileDelete;
+import static com.hedera.services.bdd.spec.transactions.TxnVerbs.systemFileDelete;
+import static com.hedera.services.bdd.spec.utilops.UtilVerbs.newKeyNamed;
+import static com.hedera.services.bdd.spec.utilops.UtilVerbs.updateLargeFile;
+import static com.hedera.services.bdd.spec.utilops.UtilVerbs.withOpContext;
+
+public class DiverseStateCreation extends HapiApiSuite {
+	private static final Logger log = LogManager.getLogger(DiverseStateCreation.class);
+
+	private static byte[] SMALL_CONTENTS;
+	private static byte[] MEDIUM_CONTENTS;
+	private static byte[] LARGE_CONTENTS;
+
+	public static final long SOMETIME = 1_635_780_626L;
+	public static final long FUSE_EXPIRY_TIME = 1_111_111L + SOMETIME;
+	public static final long MULTI_EXPIRY_TIME = 2_222_222L + SOMETIME;
+	public static final long SMALL_EXPIRY_TIME = 1_000_000L + SOMETIME;
+	public static final long MEDIUM_EXPIRY_TIME = 10_000_000L + SOMETIME;
+	public static final long LARGE_EXPIRY_TIME = 100_000_000L + SOMETIME;
+
+	public static final Integer EXPECTED_LUCKY_NO = 1326;
+
+	public static final String SMALL_FILE = "smallFile";
+	public static final String MEDIUM_FILE = "mediumFile";
+	public static final String LARGE_FILE = "largeFile";
+	public static final String FUSE_INITCODE = "fuseInitcode";
+	public static final String FUSE_BYTECODE = "fuseBytecode";
+	public static final String MULTI_INITCODE = "multiInitcode";
+	public static final String FUSE_CONTRACT = "fuseContract";
+	public static final String MULTI_CONTRACT = "multiContract";
+
+	private final Map<String, Long> entityNums = new HashMap<>();
+	private final Map<String, String> keyReprs = new HashMap<>();
+	private final Map<String, String> hexedBytecode = new HashMap<>();
+	public static final String ENTITY_NUM_KEY = "entityNums";
+	public static final String KEY_REPRS_KEY = "keyReprs";
+	public static final String HEXED_BYTECODE_KEY = "hexedBytecode";
+
+	public static final String SMALL_CONTENTS_LOC = "src/main/resource/testfiles/small.txt";
+	public static final String MEDIUM_CONTENTS_LOC = "src/main/resource/testfiles/medium.txt";
+	public static final String LARGE_CONTENTS_LOC = "src/main/resource/testfiles/large.txt";
+	public static final String STATE_META_JSON_LOC = "src/main/resource/testfiles/diverseBlobsInfo.json";
+
+	public static void main(String... args) throws IOException {
+		SMALL_CONTENTS = Files.newInputStream(Paths.get(SMALL_CONTENTS_LOC)).readAllBytes();
+		MEDIUM_CONTENTS = Files.newInputStream(Paths.get(MEDIUM_CONTENTS_LOC)).readAllBytes();
+		LARGE_CONTENTS = Files.newInputStream(Paths.get(LARGE_CONTENTS_LOC)).readAllBytes();
```

Comment:
I think this preparation step should be moved to the `getSpecsInSuite` method, otherwise if we run it with `SuiteRunner`, we won't have the fields initialized.

Reply:
Ah great point! Done. 👍